modules/ROOT/examples/live-demos/uploadcare-video/example.js

```diff
@@ -0,0 +1,38 @@
+tinymce.init({
+    selector: "textarea#uploadcare-video",
+    plugins: [ "uploadcare", "code", "link", "preview", "lists" ],
+    uploadcare_public_key: '<your-public-key>',
```

Comment:
Just want to confirm that any public key is fine

Reply:
They replace with their `uc` public key, if your suggesting that `<your-public-key>` is to generic we can update.

Reply:
I think being more specific to which public key should be used is better